kotlin/android/app/src/main/java/dev/firezone/android/tunnel/util/DnsServersDetector.kt

```diff
@@ -40,36 +40,23 @@ class DnsServersDetector(
 ) {
     //region - public //////////////////////////////////////////////////////////////////////////////
     // //////////////////////////////////////////////////////////////////////////////////////////////
-    val servers: Array<String>
+    val servers: Set<InetAddress>
```

Comment:
Good call making this a set! @pratikvelani I believe this had contained duplicate IPs.